.gitlab-ci.yml

```diff
@@ -145,15 +143,15 @@ javascript-eslint:
   before_script:
     - cd benchexec/tablegenerator/react-table
     - npm install
+    - npm run dependencies
```

Comment:
This should be run automatically if it is necessary, just like we do it for the other build jobs.

Reply:
I added a specific script to the package.json that does that